apps/docs/content/components/Text/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Text
+description: Use the text component to render body content
```

Comment:
The term "body content" generally encompasses paragraphs and headings (among other textual components) in the main text of a document. Should we perhaps be more specific in this description, as I believe the text component is not used for headings?